app/assets/v2/js/activity.js

```diff
@@ -1,4 +1,7 @@
 /* eslint no-useless-concat: 0 */ // --> OFF
+window.addEventListener('load', function() {
+  setInterval(listen_for_web3_changes, 5000);
```

Comment:
note that this wont populate the web3 info for 5000ms after pageload. listen_for_web3_changes() should probably be called immediately upon load unless we re hoping that people dont interact with web3 for 5s.